packages/app/public/index.html

```diff
@@ -48,6 +48,23 @@
       }
     </style>
     <title><%= app.title %></title>
+
+    <% if (app.googleAnalyticsTrackingId && typeof app.googleAnalyticsTrackingId
+    === 'string') { %>
```

Comment:
Believe it or not this actually compiles. 🤷 

Prettier complains if I oneline it.